uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Assignment.php

```diff
@@ -24,36 +24,26 @@ public static function create($params) {
 
 
   /**
-   * Get all cases with no end dates
+   * retrieve cases along with related contact details
    *
-   * @param array $params
+   * @param array $params accept 'sort_name' and 'limit'
```

Comment:
I'm not sure here, but I'm thinking that, instead of this $params parameter, it would be better to have two separate parameters. What do you think.

In any case, it would be good to better describe what those $params are. It's easy to get what limit does, but sort_name is not that self-descriptive.